osdk/tools/docker/Dockerfile

```diff
@@ -35,7 +35,7 @@ FROM build-qemu AS qemu
 # The libslirp dependency is for QEMU's network backend.
 WORKDIR /root
 
-RUN wget -O qemu.tar.xz https://download.qemu.org/qemu-10.0.2.tar.xz \
+RUN wget -O qemu.tar.xz https://download.qemu.org/qemu-10.1.0-rc2.tar.xz \
```

Comment:
Add a FIXME comment as a reminder to switch to the stable version of 10.1.0.

The content of `osdk/tools/docker/tdx/Dockerfile` and `tools/docker/tdx/Dockerfile` is basically building TDX-aware QEMU. Now that the upstream version of QEMU supports TDX, it seems that the `asterinas/osdk-tdx` and `asterinas/asterinas-tdx` Docker images are no longer needed. So if we are going to switch to QEMU 10.1.0-rc2 , then you should adjust the Dockerfiles and Docker build CI accordingly. And finally, bump the Docker image version.